apps/web/pages/manage.tsx

```diff
@@ -0,0 +1,980 @@
+        });
+      } else {
+        redirectToExternalUrl(urlWithSuccessParams);
+        window.clearInterval(timerRef.current as number);
+      }
+    }, 1000);
+    return () => {
+      window.clearInterval(timerRef.current as number);
+    };
+  }, [timeRemaining, urlWithSuccessParams]);
+
+  if (!isToastVisible) {
+    return null;
+  }
+
+  return (
+    <>
+      <div className="relative z-[60] pb-2 sm:pb-5">
+        <div className="mx-auto w-full sm:max-w-7xl sm:px-2 lg:px-8">
+          <div className="border border-green-600 bg-green-500 p-2 sm:p-3">
+            <div className="flex flex-wrap items-center justify-between">
+              <div className="flex w-0 flex-1 items-center">
+                <p className="truncate font-medium text-white sm:mx-3">
+                  <span className="md:hidden">Redirecting to {url} ...</span>
+                  <span className="hidden md:inline">
+                    {t("you_are_being_redirected", { url, seconds: timeRemaining })}
+                  </span>
+                </p>
+              </div>
+              <div className="order-3 mt-2 w-full flex-shrink-0 sm:order-2 sm:mt-0 sm:w-auto">
+                <button
+                  onClick={() => {
+                    redirectToExternalUrl(urlWithSuccessParams);
+                  }}
+                  className="flex w-full items-center justify-center rounded-sm border border-transparent bg-white px-4 py-2 text-sm font-medium text-green-600 hover:bg-green-50">
+                  {t("continue")}
+                </button>
+              </div>
+              <div className="order-2 flex-shrink-0 sm:order-3 sm:ml-2">
+                <button
+                  type="button"
+                  onClick={() => {
+                    setIsToastVisible(false);
+                    window.clearInterval(timerRef.current as number);
+                  }}
+                  className="-mr-1 flex rounded-md p-2 hover:bg-green-600 focus:outline-none focus:ring-2 focus:ring-white">
+                  <Icon.FiX className="h-6 w-6 text-white" />
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+type SuccessProps = inferSSRProps<typeof getServerSideProps>;
+
+const stringToBoolean = z
+  .string()
+  .optional()
+  .transform((val) => val === "true");
+
+export const querySchema = z.object({
+  uid: z.string(),
+  allRemainingBookings: stringToBoolean,
+  cancel: stringToBoolean,
+  changes: stringToBoolean,
+  reschedule: stringToBoolean,
+  isSuccessBookingPage: z.string().optional(),
+});
+
+export default function Success(props: SuccessProps) {
+  const { t } = useLocale();
+  const router = useRouter();
+
+  const {
+    allRemainingBookings,
+    isSuccessBookingPage,
+    cancel: isCancellationMode,
+    changes,
+  } = querySchema.parse(router.query);
+
+  if ((isCancellationMode || changes) && typeof window !== "undefined") {
```

Comment:
Code copied from the success.tsx file. The only changes in this file are the new URL parameter 'changes' which is set to true for the 'Need to reschedule or cancel?' Link in the calendar event which makes it scroll down automatically in smaller screens